setup.cfg

```diff
@@ -36,6 +36,14 @@ install_requires =
   scipy >= 0.19
   setuptools >= 41.2
 
+[options.packages.find]
+exclude =
+  plasmapy.examples
+  plasmapy.examples.*
+
+[options.entry_points]
+plasmapy.addons =
```

Comment:
I guess this would sort of initialize an empty list of `addons` entry point packages?

Reply:
I debated about leaving this here.  It doesn't "initialize" the `entry_point`, in fact, it does nothing.  I was originally trying to have a simple `entry_point` example here, but I haven't come up with a clean looking one, and why would you create an entry point into your own package from your package. **I kept this here because it advertises the entry point.**